<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualScanner</class>
 <widget class="QDialog" name="ManualScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Manual scanner</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="toolTip">
      <string>Close this dialog</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonScanNow">
     <property name="toolTip">
      <string>Scan manual folders now</string>
     </property>
     <property name="text">
      <string>Scan now</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="toolTip">
      <string>Scanner log</string>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pushButtonScanNow</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ManualScanner</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
